### ISSUE TITLE
Unable to run versioning multimodule example

### ISSUE THREAD

**Issue by rajdeep1008** [Kotlin/dokka#2260]
**Describe the bug**
While trying to run ./gradlew dokkaHtml or ./gradlew dokkaHtmlMultiModule task in the versioning multimodule example, it gives an error stating no repositories are defined.

**Expected behaviour**
Documentation should be generated

**Screenshots**
```
> Task :parentProject:childProjectA:dokkaHtmlPartial FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':parentProject:childProjectA:dokkaHtmlPartial'.
> Could not resolve all files for configuration ':parentProject:childProjectA:dokkaHtmlPartialPlugin'.
   > Cannot resolve external dependency org.jetbrains.dokka:dokka-base:1.6.0 because no repositories are defined.
     Required by:
         project :parentProject:childProjectA
   > Cannot resolve external dependency org.jetbrains.dokka:versioning-plugin:1.6.0 because no repositories are defined.
     Required by:
         project :parentProject:childProjectA

```
```
> Task :parentProject:dokkaHtml FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':parentProject:dokkaHtml'.
> Could not resolve all files for configuration ':parentProject:dokkaHtmlPlugin'.
   > Cannot resolve external dependency org.jetbrains.dokka:dokka-base:1.6.0 because no repositories are defined.
     Required by:
         project :parentProject
   > Cannot resolve external dependency org.jetbrains.dokka:versioning-plugin:1.6.0 because no repositories are defined.
     Required by:
         project :parentProject

```

**To Reproduce**
Simply opening the versioning multimodule example and trying to generate the documentation should be enough to reproduce the issue.

**Dokka configuration**
N/A

**Installation**
- Operating system: macOS
- Build tool: Gradle 7.3
- Dokka version: 1.6.0

**Additional context**
Solution: Adding the mavenCentral repository in the subprojects block of root level build.gradle fixed the issue for me.

```
subprojects {
    repositories {
        mavenCentral()
    }

    apply {
        plugin("org.jetbrains.kotlin.jvm")
        plugin("org.jetbrains.dokka")
    }
    val dokkaPlugin by configurations
    dependencies {
       dokkaPlugin("org.jetbrains.dokka:versioning-plugin:1.6.0")
    }
}
```

**Are you willing to provide a PR?**
If you can point me to the contribution guidelines or let me know how to go about it, i'll be more than happy to raise a PR.

**Comment by vmishenev** [comment]
Please feel free to raise a PR from your fork.

**Comment by rajdeep1008** [comment]
Hey @vmishenev 
Raised a PR to fix this #2263
